artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/QueueControlImpl.java

```diff
@@ -249,6 +249,21 @@ public long getMessageCount() {
       }
    }
 
+   @Override
+   public float getProducedRate() {
+      if (AuditLogger.isEnabled()) {
+         AuditLogger.getProducedRate(queue);
+      }
+      checkStarted();
+
+      clearIO();
```

Comment:
Shouldnt be doing this on an attribute will cause perf issues when metric systems scrape blocking deliveries

Reply:
@michaelandrepearce I think it's harmless though.